Neos.Neos/Classes/Domain/Service/WorkspaceService.php

```diff
@@ -95,9 +97,7 @@ public function setWorkspaceDescription(ContentRepositoryId $contentRepositoryId
     }
 
     /**
-     * Retrieve the currently active personal workspace for the specified $userId
-     *
-     * NOTE: Currently there can only ever be a single personal workspace per user. But this API already prepares support for multiple personal workspaces per user
+     * Retrieve the personal workspace for the specified user, if no workspace exist an exception is thrown.
```

Comment:
out of curiosity: why did you remove part of the doc comment?

Reply:
because i removed the possibility to have multiple personal workspaces (see https://github.com/neos/neos-development-collection/pull/5334#discussion_r1876050738) any future may happen ideas dont have to be commented in code.

Reply:
yep, makes sense!